### PR TITLE
Feature: allow to drag over chart

### PR DESCRIPTION
**Pull request by zefirka**

### CODE REVIEW

File: src/YagrCore/plugins/tooltip/tooltip.ts [hunk 1]
```diff
@@ -592,8 +647,14 @@ class YagrTooltip {
         this.show();
     };
 
-    private onMouseLeave = () => {
-        if (!this.state.pinned) {
+    private onMouseLeave = (e: MouseEvent) => {
+        const isPiiined = this.state.pinned;
```
Comment (feelsbadmans) [inline src/YagrCore/plugins/tooltip/tooltip.ts:651]: ```isPinned``` I guess

File: src/YagrCore/plugins/tooltip/tooltip.ts [hunk 2]
```diff
@@ -562,9 +566,60 @@ class YagrTooltip {
...
+
+            result = range[0];
+        }
+
+        this.yagr.uplot.setCursor({
+            left: result.clientX,
+            top: e.clientY - rect.top,
+        });
+
+        return result;
```
Comment (feelsbadmans) [inline src/YagrCore/plugins/tooltip/tooltip.ts:607]: It seems that we shouldn't return a value, because we don't use it

**Review by feelsbadmans — APPROVED** [review]

File: src/YagrCore/plugins/tooltip/tooltip.ts (hunk 1 above)
Comment (zefirka) [reply src/YagrCore/plugins/tooltip/tooltip.ts:651]: Yup

File: src/YagrCore/plugins/tooltip/tooltip.ts (hunk 2 above)
Comment (zefirka) [reply src/YagrCore/plugins/tooltip/tooltip.ts:607]: Totally!

**Pull request merged**